### PR TITLE
Configure artificial delay to showcase metrics

### PR DESCRIPTION
**Pull request by InTheCloudDan**
Enables enriched data export metrics by tracking how long server rendering takes.

### CODE REVIEW

File: app/util.py [hunk 1]
```diff
@@ -21,3 +23,14 @@ def getLdMachineUser(request=None):
     }
     logging.debug(user)
     return user
+
+def newServerFunctionality(duration):
+    # Delays are in seconds
+    min_load_delay = 1
```
Comment (levlaz) [inline app/util.py]: Sorry to nitpick, can you make these CONSTANTS?

File: app/util.py [hunk 2]
```diff
@@ -21,3 +23,14 @@ def getLdMachineUser(request=None):
...
+    max_load_delay = 3
+
+    # multi-variate flag is a string, so doing string comparison
+    if duration == '30':
```
Comment (levlaz) [inline app/util.py]: Same here, no magic numbers please.

File: app/routes.py [hunk 3]
```diff
@@ -11,7 +12,7 @@
 
 from app.factory import CACHE_TIMEOUT, CachingDisabled, cache, db
 from app.models import User, Plan
-
+from app.util import newServerFunctionality
```
Comment (levlaz) [inline app/routes.py]: I don't like the name of this function, can you rename it to something that means something? 

Also can we please switch to snake_case (PEP8). I know there is a mixed bag in here, but I'd like to get us over to PEP8 ASAP.

**Comment by levlaz** [comment]
This is great @InTheCloudDan, as a final step can you please squash all of this into a single commit?

**Comment by InTheCloudDan** [comment]
> This is great @InTheCloudDan, as a final step can you please squash all of this into a single commit?

@levlaz what is the reason for us to do that and not to do a `Squash and merge`?

**Comment by levlaz** [comment]
> > This is great @InTheCloudDan, as a final step can you please squash all of this into a single commit?
> 
> @levlaz what is the reason for us to do that and not to do a `Squash and merge`?

I don't want merge commits in the repo. Due to: https://chris.beams.io/posts/git-commit/

**Pull request merged**